cibuildwheel/linux.py

```diff
@@ -101,8 +103,8 @@ def build(project_dir, output_dir, test_command, test_requires, before_build, bu
                 fi
 
                 # we're all done here; move it to output
-                mv "$delocated_wheel" /output
-                chown {uid}:{gid} "/output/$(basename "$delocated_wheel")"
+                mv "${{delocated_wheels[@]}}" /output
+                for delocated_wheel in "${{delocated_wheels[@]}}"; do chown {uid}:{gid} "/output/$(basename "$delocated_wheel")"; done
```

Comment:
I think `chown` will accept multiple files, but with the basename in there, the loop is probably the right approach. Although perhaps we could just do the chown before moving to output, then the basename wouldn't be needed.

Reply:
Yeah, it's quite annoying.

I thought there was maybe a reason you first `mv`ed, then `chown`ed, though? Something that there's a chance you can't move the files anymore if they're not yours (although I guess we are root in the docker image). But then there was no reason for this order?

Anyway, no strong opinion here, so I can put whatever you prefer.